dist/shabdawali.js

```diff
@@ -221,7 +221,8 @@ Shabdawali.prototype.typeText = function(cLine){
                 this.deleteText(cLine);
             }else{
                 var char = cLine.substr( this.currentLetterIndex++ , 1);
-                this.onChar( char );
```

Comment:
This file is auto generated. So any change to this file should be avoided.

Reply:
okay, let me make new fork from your repo and upload changes in index.html again without any changes in shabdawali.js?

Reply:
yes please thanks